internal/builders/gradle/create_attestation.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
```

Comment:
Make sure to fail when any command fails.
```suggestion

set -euo pipefail

```